agenta-web/src/components/Playground/NewVariantModal.tsx

```diff
@@ -30,22 +30,33 @@ const NewVariantModal: React.FC<Props> = ({
 }) => {
     const classes = useStyles()
     const [variantPlaceHolder, setVariantPlaceHolder] = useState("Source Variant")
+    const [isInputValid, setIsInputValid] = useState(false) // To check if the input is valid
+
     const handleTemplateVariantChange = (value: string) => {
         let newValue = value.includes(".") ? value.split(".")[0] : value
         setTemplateVariantName(value)
         setVariantPlaceHolder(`${newValue}`)
     }
 
+    const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
this method name is too general. what input is this?

Reply:
> this method name is too general. what input is this?
In this context, input refers to the name of a variant. will make this clearer in the function name.

Reply:
You may create an issue for it